PF/Assignment 4/Day4Assignment.java

```diff
@@ -0,0 +1,96 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Day4Assignment
+{
+	final int NUMBER_OF_COLUMNS = 4; // for matrix of 4X4 for N Queen Problem
+	List<String> towerStrings = new ArrayList<String>();
```

Comment:
in that case output will make some issue...cant use it as instance...as we will be re initializing it again and again in case of recursion... this variable is just holding all output string while this method runs or goes into recursion... recursion will overwrite it..so made it instance var